tests/Localization/LtLtTest.php

```diff
@@ -17,210 +17,415 @@ class LtLtTest extends LocalizationTestCase
         'prieš 1 dieną',
+
         // Carbon::now()->subDays(1)->diffForHumans(null, false, true)
+        // '1d ago'
         'prieš 1 d.',
+
         // Carbon::now()->subDays(2)->diffForHumans()
+        // '2 days ago'
         'prieš 2 dienas',
+
         // Carbon::now()->subDays(2)->diffForHumans(null, false, true)
+        // '2d ago'
         'prieš 2 d.',
+
         // Carbon::now()->subWeeks(1)->diffForHumans()
+        // '1 week ago'
         'prieš 1 savaitę',
+
         // Carbon::now()->subWeeks(1)->diffForHumans(null, false, true)
+        // '1w ago'
         'prieš 1 sav.',
+
         // Carbon::now()->subWeeks(2)->diffForHumans()
+        // '2 weeks ago'
         'prieš 2 savaites',
+
         // Carbon::now()->subWeeks(2)->diffForHumans(null, false, true)
+        // '2w ago'
         'prieš 2 sav.',
+
         // Carbon::now()->subMonths(1)->diffForHumans()
+        // '1 month ago'
         'prieš 1 mėnesį',
+
         // Carbon::now()->subMonths(1)->diffForHumans(null, false, true)
+        // '1mo ago'
         'prieš 1 mėn.',
+
         // Carbon::now()->subMonths(2)->diffForHumans()
+        // '2 months ago'
         'prieš 2 mėnesius',
+
         // Carbon::now()->subMonths(2)->diffForHumans(null, false, true)
+        // '2mos ago'
         'prieš 2 mėn.',
+
         // Carbon::now()->subYears(1)->diffForHumans()
+        // '1 year ago'
         'prieš 1 metus',
+
         // Carbon::now()->subYears(1)->diffForHumans(null, false, true)
+        // '1yr ago'
         'prieš 1 m.',
+
         // Carbon::now()->subYears(2)->diffForHumans()
+        // '2 years ago'
         'prieš 2 metus',
+
         // Carbon::now()->subYears(2)->diffForHumans(null, false, true)
+        // '2yrs ago'
         'prieš 2 m.',
+
         // Carbon::now()->addSecond()->diffForHumans()
+        // '1 second from now'
         'už 1 sekundės',
+
         // Carbon::now()->addSecond()->diffForHumans(null, false, true)
+        // '1s from now'
         'už 1 sek.',
+
         // Carbon::now()->addSecond()->diffForHumans(Carbon::now())
-        'po 1 sekundę',
+        // '1 second after'
+        'po 1 sekundė',
+
         // Carbon::now()->addSecond()->diffForHumans(Carbon::now(), false, true)
+        // '1s after'
         'po 1 sek.',
+
         // Carbon::now()->diffForHumans(Carbon::now()->addSecond())
-        '1 sekundę nuo dabar',
+        // '1 second before'
+        '1 sekundė nuo dabar',
+
         // Carbon::now()->diffForHumans(Carbon::now()->addSecond(), false, true)
+        // '1s before'
         '1 sek. nuo dabar',
+
         // Carbon::now()->addSecond()->diffForHumans(Carbon::now(), true)
-        '1 sekundę',
+        // '1 second'
+        '1 sekundė',
+
         // Carbon::now()->addSecond()->diffForHumans(Carbon::now(), true, true)
+        // '1s'
         '1 sek.',
+
         // Carbon::now()->diffForHumans(Carbon::now()->addSecond()->addSecond(), true)
+        // '2 seconds'
         '2 sekundes',
```

Comment:
`2 sekundės` last `e` should be replaces by `ė`

Reply:
Fixed.